django_ipam/static/django-ipam/js/widget.js

```diff
@@ -0,0 +1,18 @@
+/*jslint browser: true*/
+/*global $, jQuery, alert*/
+
+function getAvailableIp() {
+    "use strict";
+    var e = document.getElementById("id_subnet");
+    var subnet = e.options[e.selectedIndex].value;
+    $.ajax({
+        type: 'GET',
+        url: "http://" + window.location.host + "/api/v1/subnet/" + subnet + "/get-first-available-ip",
+        success: function (res) {
+            if (res === '') {
```

Comment:
are you sure this works? The response in this case returns `None` which would be serialized to `null` and `null === ''` is `false`, so I think this wouldn't work

Reply:
I am using `''` because `Response(None)` return an empty body response, instead of sending a serialized `null`.